mrtt-ui/src/components/SiteInterventions.js

```diff
@@ -0,0 +1,162 @@
+  Form
+} from '../styles/forms'
+import { ErrorText, Link } from '../styles/typography'
+import language from '../language'
+import { questionMapping } from '../data/questionMapping'
+import { siteInterventions as questions } from '../data/questions'
+import { mapDataForApi } from '../library/mapDataForApi'
+import { ButtonSubmit } from '../styles/buttons'
+import { multiselectWithOtherValidation } from '../validation/multiSelectWithOther'
+import useInitializeQuestionMappedForm from '../library/useInitializeQuestionMappedForm'
+import LoadingIndicator from './LoadingIndicator'
+import CheckboxGroupWithLabelAndController from './CheckboxGroupWithLabelAndController'
+
+function SiteInterventionsForm() {
+  const validationSchema = yup.object({
+    whichStakeholdersInvolved: multiselectWithOtherValidation,
+    biophysicalInterventionsUsed: multiselectWithOtherValidation,
+    localParticipantTraining: yup.string(),
+    organizationsProvidingTraining: multiselectWithOtherValidation,
+    otherActivitiesImplemented: multiselectWithOtherValidation
+  })
+  const reactHookFormInstance = useForm({
+    defaultValues: {
+      whichStakeholdersInvolved: { selectedValues: [], otherValue: undefined },
+      biophysicalInterventionsUsed: { selectedValues: [], otherValue: undefined },
+      organizationsProvidingTraining: { selectedValues: [], otherValue: undefined },
+      otherActivitiesImplemented: { selectedValues: [], otherValue: undefined }
+    },
+    resolver: yupResolver(validationSchema)
+  })
+
+  const {
+    handleSubmit: validateInputs,
+    formState: { errors },
+    reset: resetForm,
+    watch: watchForm,
+    control
+  } = reactHookFormInstance
+
+  const { siteId } = useParams()
+  const apiAnswersUrl = `${process.env.REACT_APP_API_URL}/sites/${siteId}/registration_answers`
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const [isSubmitError, setIsSubmitError] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
+  const localParticipantTrainingWatcher = watchForm('localParticipantTraining')
+
+  useInitializeQuestionMappedForm({
+    apiUrl: apiAnswersUrl,
+    questionMapping: questionMapping.siteInterventions,
+    resetForm,
+    setIsLoading
+    // successCallback: loadStakeholdersFromServerData
+  })
+
+  const handleSubmit = (formData) => {
+    setIsSubmitting(true)
+    setIsSubmitError(false)
+
+    axios
+      .patch(apiAnswersUrl, mapDataForApi('siteInterventions', formData))
+      .then(() => {
+        setIsSubmitting(false)
+        toast.success(language.success.submit)
+      })
+      .catch(() => {
+        setIsSubmitting(false)
+        setIsSubmitError(true)
+        toast.error(language.error.submit)
+      })
+  }
+  return isLoading ? (
+    <LoadingIndicator />
+  ) : (
+    <ContentWrapper>
+      <FormPageHeader>
+        <SectionFormTitle>Site Interventions</SectionFormTitle>
+        <Link to={-1}>&larr; {language.form.navigateBackToSiteOverview}</Link>
```

Comment:
Im starting to wonder in the -1 we've been putting is the best idea. For instance with the new form nav, I could have gotten here from another form section. Maybe put in the path to the site overview.

Im getting rid of my -1 navs when I bump into them

Reply:
I think hiving the path to site overview is a better idea